ocfweb/static/scss/pages/docs/_servers.scss

```diff
@@ -69,6 +69,13 @@
                     }
                 }
             }
+
+            .server-anchor {
+                display: block;
+                position: relative;
+                top: -75px;
```

Comment:
The 75px was just a number I picked while testing and it happened to work nicely on my screen but obviously doesn't work with the responsive layout. I've copied that code from _docs.scss into my .server-anchors class but not knowing sass too well I don't know if there's a more DRY way of doing it